### PR TITLE
agora UI improvements!

### PR DESCRIPTION
**Pull request by Hamzaalam**
Minor UI improvements!
Include editor colors for dark mode! (still investigating why it's working!)
Screenshot from 2022-09-22 02-28-47
Screenshot from 2022-09-22 02-28-31

### CODE REVIEW

**Review by Billalkhn — APPROVED** [review]

**Comment by Fahad-Mahmood** [comment]
@Hamzaalam  please attach screenshots for improvements in the PR.

**Review by Billalkhn — APPROVED** [review]

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "presets": ["next/babel"],
+      "plugins": [["styled-components", { "ssr": true }]]
+
+    },
+    "test": {
+      "presets": ["next/babel","@babel/preset-typescript"],
+      "plugins": ["styled-jsx/babel-test"]
+    }
+  }
+}
```
Comment (Fahad-Mahmood) [inline .babelrc:18]: we don't need this for styled-components after recent nextjs update, we can continue using swc compiler, it's 17x faster than babel.

File: styles/theme.ts [hunk 2]
```diff
@@ -13,7 +13,10 @@ export const lightTheme = {
     background2: '#f4f6f8',
     primary: '#ffffff',
     secondary: '#0221ba',
+    text: '#000000',
     text3: '#000',
+    link: '#050b21',
+    codeblock: '#050b210d',
```
Comment (Fahad-Mahmood) [inline styles/theme.ts:19]: are we using these newly added colors somewhere?

**Review by Fahad-Mahmood — COMMENTED** [review]
Thread UI in the main branch is something like this, did we add sidebar border and changed the editor text to white in this PR?

Screenshot 2022-09-22 at 10 53 11 PM

**Comment by Fahad-Mahmood** [comment]
Yeah looks good, I was confused because the screenshot had black text color which doesn't look good.

**Review by Fahad-Mahmood — APPROVED** [review]

**Pull request merged**